plugins/hydra_nevergrad_sweeper/hydra_plugins/hydra_nevergrad_sweeper/__init__.py

```diff
@@ -0,0 +1,4 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+from .core import NevergradSweeper
+
+__all__ = ["NevergradSweeper"]
```

Comment:
Please remove the imports here. this are causing the plugin to be imported twice during discovery.
(you will need to tweak imports and class name in the config to use the full name of the NevergradSweeper).

Reply:
should be done now